packages/@aws-cdk/aws-apigateway/lib/methodresponse.ts

```diff
@@ -0,0 +1,21 @@
+export interface MethodResponse {
+
+    /**
+     * The method response's status code, which you map to an IntegrationResponse.
+     */
+    statusCode: string;
+
+    /**
+     * Response parameters that API Gateway sends to the client that called a method.
+     * Specify response parameters as key-value pairs (string-to-Boolean maps), with
+     * a destination as the key and a Boolean as the value. Specify the destination
+     * using the following pattern: method.response.header.name, where the name is a
+     * valid, unique header name. The Boolean specifies whether a parameter is required.
+     */
+    responseParameters?: { [destination: string]: boolean };
+
+    /**
+     * The method response's status code, which you map to an IntegrationResponse.
+     */
+    responseModels?: { [destination: string]: string };
```

Comment:
If we're going to implement L2 support for method responses, I think we should also support Model, since the value of each key here should be a reference to https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-model.html

Reply:
Agreed.